galsim/celestial.py

```diff
@@ -582,6 +585,40 @@ def galactic(self, epoch=2000.):
 
         return (el, b)
 
+    def ecliptic(self):
+        """Get the longitude and latitude in ecliptic coordinates corresponding to this position.
+
+        The formulae for this are quite straightforward.  It requires just a single parameter for
+        the transformation, the obliquity of the ecliptic (the Earth's axial tilt).  This routine
+        gives results that agree with a pre-existing online calculator,
+        http://lambda.gsfc.nasa.gov/toolbox/tb_coordconv.cfm, to very good precision, but it's not
+        perfect.  Based on the pattern of the errors, it seems that that converter has a more
+        precise value for the axial tilt.  However, this routine is not currently being used for
+        really precise calculations, so a percent error is tolerable.  If you need really high
+        accuracy then this routine would require improvements (not just higher precision on the
+        axial tilt but also including its very slight time dependence).
+
+        @returns the longitude and latitude as a tuple (lambda, beta), given as Angle instances.
+        """
+        import math
+
+        # We are going to work in terms of the (x, y, z) projections.
+        self._set_aux()
+
+        ep = 23.4*galsim.degrees # obliquity of the ecliptic, epsilon
```

Comment:
It seems worth being more precise here.  At least set up the syntax to take an `epoch` parameter, like we do for galactic coordinates.  

Wikipedia has a formula for epsilon [here](http://en.wikipedia.org/wiki/Ecliptic#Obliquity_of_the_ecliptic) that should be more than adequate for any conceivable use case.